the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+        tableView.register(UINib(nibName: "SwitchTableViewCell", bundle: nil), forCellReuseIdentifier: "SwitchCell")
+    }
+    
+    // MARK: - Interface Methods
+    
+    @IBAction func doneTapped(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: - Table view configuration
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 4
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return "Account"
+            
+        case 1:
+            return "Notifications"
+            
+        case 2:
+            return "Info"
+            
+        case 3:
+            return "Extra"
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            return 1
+            
+        case 1:
+            return 1
+            
+        case 2:
+            return 4
+            
+        case 3:
+            return 2
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "BaseCell", for: indexPath)
+        
+        let thirdSectionLabels =  ["The Blue Alliance Website",  "The Blue Alliance for iOS is open source", "Contributors", "View changelog"]
+        let lastSectionLabels = ["Delete network cache", "Delete app data"]
+        
+        switch indexPath.section {
+        case 0:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchCell", for: indexPath) as! SwitchTableViewCell
+            cell.populate(withTitle: "Enable myTBA")
+            
+            // Configure the cell's switch state depending on wether the user is connected or not
+            if MyTBA.shared.authentication != nil {
+                cell.switchToggle.isOn = true
+            } else {
+                cell.switchToggle.isOn = false
+            }
+            
+            return cell
+            
+        case 1:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchCell", for: indexPath) as! SwitchTableViewCell
+            cell.populate(withTitle: "Enable notifications")
+            
+            return cell
+
+        case 2:
+            cell.textLabel?.text = thirdSectionLabels[indexPath.row]
+            
+        case 3:
+            cell.textLabel?.text = lastSectionLabels[indexPath.row]
+            
+            cell.textLabel?.textColor = .red
+            cell.textLabel?.textAlignment = .center
```

Comment:
The center alignment on the text looks fine for these - the problem is the cell separator offset. Right now we still have that separator offset from the leading side and not on the trailing side. It makes these buttons look weird if we have the text centered. Let's either left align the text, or remove the separator offset from the left side